giacomo_data/jackPapa/agents/jackPapa$.ttl

```diff
@@ -0,0 +1,13 @@
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
```

Comment:
Shouldn't be here. This structure is already in profile/card$.ttl where it can be discovered by other social agents.